packages/wallets/src/features/cfd/flows/MT5/AddedMT5AccountsList/AddedMT5AccountsList.tsx

```diff
@@ -33,21 +36,40 @@ const MT5AccountIcon: React.FC<TProps> = ({ account }) => {
 const AddedMT5AccountsList: React.FC<TProps> = ({ account }) => {
     const { data: activeWallet } = useAuthorize();
     const history = useHistory();
+    const { data: poaStatus } = usePOA();
     const { title } = MarketTypeDetails[account.market_type || 'all'];
+    const { show } = useModal();
+
+    const verificationDocs = getJurisdictionContents()[account?.landing_company_short || 'svg']?.verificationDocs;
+    const shouldRequireNameAndAddress =
+        verificationDocs && account?.market_type !== 'all'
+            ? verificationDocs?.[account.market_type || 'financial']?.some(doc => doc === 'nameAndAddress')
+            : false;
+
+    const isFailedAuthentication = () => {
+        if (shouldRequireNameAndAddress) return poaStatus?.is_need_submission;
+        return false;
```

Comment:
Check only for POA statuses, later I'll include checks for POI, POI and POA statuses